js/views/userPageVw.js

```diff
@@ -812,6 +839,17 @@ module.exports = baseVw.extend({
     }
   },
 
+  renderReviews: function (model) {
+    model = model || [];
+    this.reviewsVw = new reviewsView({
```

Comment:
renderFollowing should clear out the old view, though currently I don't think there's a situation where more than one would be created.

renderFollowers works a bit differently, and will only generate one view, if the view already exists it appends new data to the collection, as part of lazy loading.